packages/select/package.json

```diff
@@ -36,7 +36,7 @@
 	"peerDependencies": {
 		"@availity/api-axios": "^3.0.1",
 		"@availity/api-core": "^3.0.0",
-		"@availity/form": "^0.1.0",
+		"@availity/form": "^0.1.1",
```

Comment:
There is no `0.1.1` next lowest version if `0.2.0`

Reply:
@GoPro16, I just changed it to `^0.2.0`